src/client/components/MyProfile/YourUploadedCVs/RatingStars.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Star } from './Star';
+
+export function RatingStars({ averageStars }) {
```

Comment:
@valmdz I think it is because of proptypes. I did not define proptypes as it was a recommended part of task and I did not do it. The code working good.

Reply:
However I did it. :)